mobile/lib/common/domain/background_task.dart

```diff
@@ -49,3 +49,17 @@ class Rollover {
     return bridge.BackgroundTask_Rollover(TaskStatus.apiDummy());
   }
 }
+
+class RecoverDlc {
```

Comment:
is this only used for the purpose of opening / closing a dialog? 

perhaps we could call it more specifically, like `RecoverDlcStatusDialog` or something like that?

Reply:
At the moment, yes there is only one consumer being the UI. The event still has a general purpose though. The sender of an event should not care about the receiver.